### PR TITLE
Match AmcExi2Stubs & odenotstub

### PR DESCRIPTION
**Pull request by Yanis42**
also adds splits for the 3 debugger files on ce-u, the ``AmcExiError`` enum is from pikmin2

### CODE REVIEW

File: src/debugger/AmcExi2Stubs.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include "debugger.h"
+
+void EXI2_Init(void) {}
+
+void EXI2_EnableInterrupts(void) {}
+
+s32 EXI2_Poll(void) { return 0; }
```
Comment (cadmic) [inline src/debugger/AmcExi2Stubs.c:7]: should `EXI2_Poll`, `AMC_IsStub`, and `Hu_IsStub` return bool?

**Review by cadmic — COMMENTED** [review]
phew, these files looked difficult

File: src/debugger/AmcExi2Stubs.c (hunk 1 above)
Comment (Yanis42) [reply src/debugger/AmcExi2Stubs.c:7]: I was wondering the same thing

**Comment by Yanis42** [comment]
probably my biggest challenge on this decomp so far

**Pull request merged**